Godeps/Godeps.json

```diff
@@ -2,371 +2,18 @@
 	"ImportPath": "github.com/coreos/rkt",
 	"GoVersion": "go1.5",
 	"Packages": [
-		"./...",
-		"github.com/appc/spec/actool",
-		"github.com/appc/cni/plugins/ipam/dhcp",
-		"github.com/appc/cni/plugins/ipam/host-local",
-		"github.com/appc/cni/plugins/main/bridge",
-		"github.com/appc/cni/plugins/main/ipvlan",
-		"github.com/appc/cni/plugins/main/macvlan",
-		"github.com/appc/cni/plugins/main/ptp",
-		"github.com/appc/cni/plugins/meta/flannel",
-		"github.com/appc/spec/ace"
```

Comment:
Please refer to godep guide in Documentation/hacking.md. Especially the part about saving - https://github.com/coreos/rkt/blob/master/Documentation/hacking.md#saving-the-set-of-dependencies